docs/guides/fractional-gates.ipynb

```diff
@@ -0,0 +1,378 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "116af785-a9b7-4eaf-8a86-b1eeb235da18",
+   "metadata": {},
+   "source": [
+    "# Fractional gates\n",
+    "\n",
+    "This page introduces two newly supported gate types on the IBM Quantum&trade; fleet of QPUs. These *fractional* gates are supported on [Heron QPUs](/guides/processor-types#heron) in the form of:\n",
+    "- $R_{ZZ}(\\theta)$ for $0 \\lt \\theta \\leq \\pi/2$\n",
+    "- $R_X(\\theta)$ for any $\\theta$\n",
+    "\n",
+    "This page discusses the use cases where implementing fractional gates can improve the efficiency of your workflows, as well as how to use these gates on IBM Quantum QPUs."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "ee241e99-d095-4cb5-9506-da3d92c32e1c",
+   "metadata": {},
+   "source": [
+    "## How to use fractional gates\n",
+    "\n",
+    "Internally, these fractional gates work by directly executing an $R_{ZZ}(\\theta)$ and $R_X(\\theta)$ rotation for an arbitrary angle. Use of the $R_X(\\theta)$ gate can reduce the duration and error for single-qubit rotations of arbitrary angle by up to a factor of two. The direct execution of the $R_{ZZ}(\\theta)$ gate rotation avoids decomposition into multiple `CZGate`s, similarly reducing a circuit's duration and error. This is especially useful for circuits that contain many single- and two-qubit rotations, such as when simulating the dynamics of a quantum system or when using a variational ansatz with many parameters.\n",
+    "\n",
+    "\n",
+    "While these types of gates are in the [library of standard gates](./circuit-library) which a `QuantumCircuit` can possess, they can only be used on specific IBM Quantum QPUs which must be loaded with the flag `use_fractional_gates` set to `True` (shown below). This flag will ensure that fractional gates are included in the backend's `Target` for the transpiler.\n",
+    "\n",
+    "```python\n",
+    "service = QiskitRuntimeService()\n",
+    "backend = service.backend('ibm_torino', use_fractional_gates=True)\n",
+    "```\n",
+    "\n",
+    "This code example demonstrates how to use fractional gates in the context of a workflow that simulates the dynamics of an Ising chain using fractional gates. The circuit duration is then compared against a backend that does not use fractional gates.\n",
+    "\n",
+    "<Admonition title=\"Note about reported error rates\" type=\"note\">\n",
+    "  The error value reported in the `Target` of a backend with fractional gates enabled is just a copy of the non-fractional gate's counterpart (which may not be the same). This is because the reporting of error rates on the fractional gates is not yet supported.\n",
+    "\n",
+    "  However, since the gate time of fractional vs non-fractional gates are the same, it is a reasonable assumption that their error rates are comparable -- especially when the dominant source of error in a circuit is due to relaxation.\n",
```

Comment:
```suggestion
    "  However, since the gate time of fractional versus non-fractional gates are the same, it is a reasonable assumption that their error rates are comparable -- especially when the dominant source of error in a circuit is due to relaxation.\n",
```